introduction.md

```diff
@@ -0,0 +1,2 @@
+My name is Doru. I am studying github workflow.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.